cmd/server/assets/login/account.html

```diff
@@ -0,0 +1,115 @@
+{{define "account"}}
+
+{{$user := .currentUser}}
+
+<!doctype html>
+<html lang="en">
+
+<head>
+  {{template "head" .}}
+  {{template "firebase" .}}
+</head>
+
+<body class="bg-light">
+  {{template "navbar" .}}
+
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <h1>My Account</h1>
+    <p>Information and settings for your account</p>
+
+    <div class="card mb-3 shadow-sm">
+      <div class="card-header">Details</div>
+      <div class="card-body">
+        <h6 class="card-title">Name</h6>
+        <div class="card-text mb-3">
+          {{$user.Name}}
+        </div>
+
+        <h6 class="card-title">Email</h6>
+        <div class="card-text  mb-3">
+          {{$user.Email}}
+        </div>
+
+        <h6 class="card-title">System admin</h6>
```

Comment:
I don't think we should disclose to average users that system admins exist. Perhaps wrap this entire thing in an `{{if $user.Admin}}`?

Reply:
And then if you switch the CSS to mt instead of mb, you avoid the edge flowing over on the last element